src/components/feed-sources/feed-source-type-templates/EventDatabaseApiFeedType.template.jsx

```diff
@@ -0,0 +1,37 @@
+import { React } from "react";
+import PropTypes from "prop-types";
+import FormInput from "../../util/forms/form-input";
+
+const EventDatabaseApiTemplate = ({
+  handleInput,
+  formStateObject,
+  t,
```

Comment:
no need to inject t through props...

const { t } = useTranslation("common", { keyPrefix: "event-database-feed-type" });